src/herbpy/action/pouring.py

```diff
@@ -0,0 +1,104 @@
+import openravepy, os, time, prpy, actions, math
+from prpy.action import ActionMethod
+from herbpy.tsr.cup import *
+from herbpy.tsr.pitcher import *
+
+def PourWater(robot, cup, pitcher):
```

Comment:
This function changes the robot's active manipulator, active DOFs, and velocity limits. You should wrap the relevant portions of the function in a robot state saver to restore them to the defaults before returning.